Source/SwiftLintFramework/Rules/NimbleOperatorRule.swift

```diff
@@ -60,4 +71,18 @@ public struct NimbleOperatorRule: ConfigurationProviderRule, OptInRule {
                 location: Location(file: file, characterOffset: $0.0.location))
         }
     }
+    public func correctFile(_ file: File) -> [Correction] {
+        let variable = "\\s*(.*?)\\s*"
+        let patterns = [
+            "expect\\(\(variable)\\)\\.to\\(equal\\(\(variable)\\)\\)": "expect($1) == $2",
+            "expect\\(\(variable)\\)\\.toNot\\(equal\\(\(variable)\\)\\)": "expect($1) != $2",
+            "expect\\(\(variable)\\)\\.to\\(beIdenticalTo\\(\(variable)\\)\\)": "expect($1) === $2",
+            "expect\\(\(variable)\\)\\.toNot\\(beIdenticalTo\\(\(variable)\\)\\)": "expect($1) !== $2",
+            "expect\\(\(variable)\\)\\.to\\(beGreaterThan\\(\(variable)\\)\\)": "expect($1) > $2",
+            "expect\\(\(variable)\\)\\.to\\(beGreaterThanOrEqualTo\\(\(variable)\\)\\)": "expect($1) >= $2",
+            "expect\\(\(variable)\\)\\.to\\(beLessThan\\(\(variable)\\)\\)": "expect($1) < $2",
+            "expect\\(\(variable)\\)\\.to\\(beLessThanOrEqualTo\\(\(variable)\\)\\)": "expect($1) <= $2"
+        ]
+        return file.correctLegacyRule(self, patterns: patterns)
```

Comment:
If we reuse the `File.correctLegacyRule(_:patterns:)` function, we should rename it to something more appropriate.

Also, this function validates different syntax kinds than `validateFile(_:)` does:

```swift
return matchPattern(pattern).filter { range, kinds in
    return kinds.first == .identifier &&
        !ruleEnabledViolatingRanges([range], forRule: rule).isEmpty
}...
```

This needs to be aligned so that we don't accidentally correct things that wouldn't have triggered a lint violation, or miss any.